app/views/decks/_deck_summary.html.erb

```diff
@@ -0,0 +1,106 @@
+                <td>
+                  <ul class="list-inline">
+                    <% @categories.each do |category| %>
+                      <li class="list-item-label">
+                        <span class="label label-default"><%= @category_deck_links.where(category_id: category.id).count %> <%= category.name %></span>
+                      </li>
+                    <% end %>
+                  </ul>
+                </td>
+              </tr>
+
+            <% elsif resource == @cards %>
+              
+              <tr>
+                <th><span class="glyphicon glyphicon-calendar" aria-hidden="true"></span>Created At</th>
+                <td>3/20/2016</td>
+                <td></td>
+              </tr>
+              <tr>
+                <th><span class="glyphicon glyphicon-book" aria-hidden="true"></span>Cards</th>
+                <td><%= resource.count %></td>
+                <td>
+                  <ul class="list-inline">
+                    <a href="#"><li class="list-item-label"><span class="label label-needs-review">0 Needs Review</span></li></a>
+                    <a href="#"><li class="list-item-label"><span class="label label-published">0 Published</span></li></a>
+                    <a href="#"><li class="list-item-label"><span class="label label-unpublished">0 Unpublished</span></li></a>
+                  </ul>
+                </td>
+              </tr>
+              <tr>
+                <th><span class="glyphicon glyphicon-bookmark" aria-hidden="true"></span>Categories</th>
+                <td>2</td>
+                <td>
+                  <ul class="list-inline">
+                    <li class="list-item-label"><span class="label label-default">25 Category One</span></li>
+                    <li class="list-item-label"><span class="label label-default">25 Category Two</span></li>
+                  </ul>
+                </td>
+              </tr>
+              <tr>
+                <th><span class="glyphicon glyphicon-tags" aria-hidden="true"></span>Question Type</th>
+                <td></td>
+                <td>
+                  <ul class="list-inline">
+                    <% @card_types.each do |type| %>
+                      <li class="list-item-label">
+                        <span class="label label-default"><%= resource.where(card_type: type[1]).count %> <%= type[0].humanize %></span>
+                      </li>
+                    <% end %>
+                  </ul>
+                </td>
+              </tr>
+              <tr>
+                <th><span class="glyphicon glyphicon-signal" aria-hidden="true"></span>Difficulty</th>
+                <td>7.43 Avg</td>
+                <td>
+                  <div class="progress progress-difficulty">
+                    <% for i in 0..10 %>
+                      <div class="progress-bar progress-bar-difficulty" style="width:<%= resource.where(difficulty: i).count.fdiv(resource.count) * 100 %>%">
```

Comment:
The ruby is complex enough in this line to warrant a view helper. This would also allow you to break it into two lines if you wanted / found it more readable. Also is there a strong benefit to using `.fdiv` over a regular old `/` divide? Might also be easier to read as an equation